scRNA-seq-advanced/02-celltype_assignment.Rmd

```diff
@@ -197,16 +286,32 @@ adt_df <- adt_df %>%
   )
 ```
 
-```{r}
+Now we will want to add the cell types we have assigned back to our original SCE object.
+We can do that by defining a new column name, `threshold_celltype` that will be added to the `colData` object.
+Creating and assigning values to this column can be done with the `$` shortcut, and then we can plot our results with the `plotUMAP()` function as before.
+
+```{r plot thresholds}
 sce$threshold_celltype <- adt_df$celltype
 scater::plotUMAP(sce, 
-                 colour_by = "threshold_celltype")
+                 colour_by = "threshold_celltype") + 
+  guides(color = guide_legend(title = "Cell type"))
 ```
 
-We could do similar things for any marker gene from the expression data as well.
+How did we do? 
+
+Note that while we applied this technique to assign cell types using the ADT data, we could use the same type of procedure using gene expression data alone, or a combination of gene expression data and tag data.
+
+However, what we did here was very ad-hoc and quite manual!
+We didn't calculate any statistics, and we had to look at every tag we were interested in to pick thresholds.
+A different data set might have different background levels, which would require different thresholds. 
```

Comment:
This is the first time "background levels" as a term is used. We should introduce it with the bimodal introduction (explicit BG/FG designations; I commented above as well).

Reply:
Hm also, we could explore here "what if we had chosen different thresholds" and how would our results change? We could do something like count the number of cells per cell type (in particular the unclassified!) and see the difference.

Reply:
This whole section is a "you could, but probably don't," so I don't want to spend too much time on it!